src/Exceptionless.Core/Repositories/Configuration/ExceptionlessElasticConfiguration.cs

```diff
@@ -43,7 +47,7 @@ public sealed class ExceptionlessElasticConfiguration : ElasticConfiguration, IS
             builder.Register(new StackQueryBuilder());
         }
 
-        public Settings Settings { get; }
+        public IOptionsSnapshot<ElasticsearchOptions> ElasticsearchOptions { get; }
```

Comment:
Let's rename this to just Options. It's already inside of a class that is Elasticsearch specific.

Reply:
ok